source/OutfitterPanel.cpp

```diff
@@ -321,6 +321,10 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 	if(!planet || !selectedOutfit)
 		return false;
 
+	// Add system to accumulate reasons why an outfit cannot be bought
+	vector<string> errors;
+	auto addError = [&](const string &msg) { errors.push_back(msg); };
```

Comment:
Do we really need this lambda? It just forwards the argument to push_back.

Reply:
I felt it was clearer to read, but I have now removed it.